docs/docs/gatsby-image-api.md

```diff
@@ -0,0 +1,344 @@
+    file(relativePath: { eq: "images/default.jpg" }) {
+      childImageSharp {
+        # Specify a fixed image and fragment
+        // highlight-start
+        fixed {
+          ...GatsbyImageSharpFixed
+        }
+        // highlight-end
+      }
+    }
+  `)
+  return (
+    <div>
+      <h1>Hello gatsby-image</h1>
+      <Img
+        fixed={data.file.childImageSharp.fixed} {/* highlight-line */}
+        alt="Gatsby Docs are awesome"
+      />
+    </div>
+  )
+}
+```
+
+Read more in the [gatsby-plugin-sharp](/packages/gatsby-plugin-sharp/?=#fixed) README.
+
+#### Fixed image query parameters
+
+In a query, you can specify options for fixed images.
+
+- `width` (int, default: 400)
+- `height` (int)
+- `quality` (int, default: 50)
+
+#### Returns
+
+- `base64` (string)
+- `aspectRatio` (float)
+- `width` (float)
+- `height` (float)
+- `src` (string)
+- `srcSet` (string)
+
+This is where fragments like `GatsbyImageSharpFixed` come in handy, as they'll return all the above items in one line:
+
+```graphql
+file(relativePath: { eq: "images/default.jpg" }) {
+  childImageSharp {
+    # Specify a fixed image and fragment
+    // highlight-next-line
+    fixed(width: 400, height: 400) {
+      ...GatsbyImageSharpFixed
+    }
+  }
+}
+```
+
+### Images that stretch across a _fluid_ container
+
+Create fluid sizes (in width) for the image. E.g. if the max width of the container for a rendered Markdown file is 800px, the automatic sizes would then be: 200px, 400px, 800px, 1200px, 1600px – enough to provide close to the optimal image size for every device size / screen resolution. If you want more control over which sizes are output you can use the `srcSetBreakpoints` parameter.
+
+Once you've queried for an image to retrieve its data, you can pass the `fluid` data into the `Img` component:
+
+```jsx
+import { useStaticQuery } from "gatsby"
+import Img from "gatsby-image"
+
+export default () => {
+  const data = useStaticQuery(query`
+    file(relativePath: { eq: "images/default.jpg" }) {
+      childImageSharp {
+        # Specify a fluid image and fragment
+        // highlight-start
+        fluid {
+          ...GatsbyImageSharpFluid
+        }
+        // highlight-end
+      }
+    }
+  `)
+  return (
+    <div>
+      <h1>Hello gatsby-image</h1>
+      <Img
+        fluid={data.file.childImageSharp.fluid} {/* highlight-line */}
+        alt="Gatsby Docs are awesome"
+      />
+    </div>
+  )
+}
+```
+
+Read more in the [gatsby-plugin-sharp](/packages/gatsby-plugin-sharp/?=#fluid) README.
+
+#### Fluid image query parameters
+
+In a query, you can specify options for fluid images.
+
+- `maxWidth` (int, default: 800)
+- `maxHeight`(int)
+- `quality` (int, default: 50)
+- `srcSetBreakpoints` (array of int, default: [])
+- `fit` (string, default: `[sharp.fit.cover][6]`)
+- `background` (string, default: `rgba(0,0,0,1)`)
+
+#### Returns
+
+- `src` (string)
+- `width` (int)
+- `height` (int)
+- `aspectRatio` (float)
```

Comment:
This list is missing `base64`, `srcSet`, etc.

Reply:
nice catch!